staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2502,6 +2502,16 @@ type KubeVirtConfiguration struct {
 	// +nullable
 	// +kubebuilder:validation:Enum=Stage;LiveUpdate
 	VMRolloutStrategy *VMRolloutStrategy `json:"vmRolloutStrategy,omitempty"`
+
+	// CommonInstancetypesDeployment controls the deployment of common-instancetypes resources
+	// +nullable
+	CommonInstancetypesDeployment *CommonInstancetypesDeployment `json:"commonInstancetypesDeployment,omitempty"`
```

Comment:
Although the design proposal for configurables is still under review, we have reached an agreement to place configurables inside `.spec.` directly, see: https://github.com/kubevirt/community/pull/316#discussion_r1740975240

Reply:
Okay I can move it, I'll also comment on the DP as I wonder what this means for everything already under `.configuration`?

Reply:
The discussion converged to move all fields away from `spec.configuration` to `spec`: https://github.com/kubevirt/community/pull/316#discussion_r1741583670

Reply:
Actually reading through the comments it looks like that wasn't fully agreed on and has been deferred to another design proposal? I'm not sure that I'm on board with the direction as well given the huge amount of churn for existing configurables. I might hold off on moving this for now.